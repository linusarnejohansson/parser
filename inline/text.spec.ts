import { Error, IO } from "@cogneco/mend"
import { Source } from "../Source"
import { parse } from "."

describe("inline.text", () => {
	it("basic", () => {
		const result = parse(new Source(IO.StringReader.create("A simple text."), new Error.ConsoleHandler())) || []
		expect(result.map(node => node.toObject())).toEqual([{ class: "Inline.Text", value: "A simple text." }])
	})
})
