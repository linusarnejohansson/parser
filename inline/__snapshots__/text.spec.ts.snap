// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`inline.text basic 1`] = `
Array [
  Object {
    "class": "Inline.Text",
    "value": "A simple text.",
  },
]
`;
